scripts/build/builders/openiotsdk.py

```diff
@@ -72,6 +77,12 @@ def _build(self):
         self._Execute(['bash', '-c', cmd],
                       title='Remove old artifacts ' + self.identifier)
 
+        # Install the required Python packages for TF-M tools used during its components build
+        tfm_requirements_path = os.path.join(self.output_dir, "_deps", "tf-m-src", "tools", "requirements.txt")
+        if os.path.exists(tfm_requirements_path):
+            self._Execute(['pip3', 'install', '-r', tfm_requirements_path],
```

Comment:
This will change the global build environment for the user ... so sideffects outside the build directory for the user.

What are these requirements? Could they be installed as part of general chip requirements?

Reply:
Agree, we can use Python virtual env in this case.
These are the required package for the current version:
```
cbor2
click
cryptography
pyasn1
imgtool>=1.6.0
Jinja2>=2.10.3
PyYAML
pyhsslms
ecdsa
kconfiglib
```
But it could be different depending on the TF-M version, so I preferred to install it dynamically during the build process.

Reply:
I checked the required packages and only imgtool was missed in general chip requirements. I'll at it and it should work without the additional steps during the build process.